packages/ods/src/style/_message.scss

```diff
@@ -0,0 +1,116 @@
+@mixin ods-message() {
+  display: grid;
+  grid-template-rows: min-content;
```

Comment:
I had to add this to prevent the slot content from centering vertically within its parent grid when a height is set by the user. Set to min-content will keep everything aligned at the top since it adjusts to the minimum required to fit its content.